### PR TITLE
use correct font for checkmark character

### LINKED ISSUE #9055: Character missing from font
https://github.com/wesnoth/wesnoth/blob/6812eac403982f82bdc4be0a4cf6c6b8cf235484/data/gui/widget/button_success.cfg#L250

The character ✔ is not in the used font. It's in all Dejavu Sans variants we ship though.

### PR DESCRIPTION
**Pull request by babaissarkar**
Resolves #9055.

### CODE REVIEW

File: data/gui/widget/button_success.cfg [hunk 1]
```diff
@@ -247,7 +247,7 @@
 					({GUI__BORDER_COLOR      ALPHA={ALPHA}})
 					({GUI__BORDER_COLOR_DARK ALPHA={ALPHA}}) ("21, 79, 109, 255") {IPF}}
 
-				{_GUI_BUTTON_TEXT (_"page^✔ Copied") ({FONT_SIZE}) () ("0, 255, 0, 255")}
+				{_GUI_BUTTON_TEXT (_"page^<span font_family='DejaVu Sans'>✔</span> Copied") ({FONT_SIZE}) () ("0, 255, 0, 255")}
```
Comment (stevecotton) [inline data/gui/widget/button_success.cfg:250]: This needs a `#po:` explanation to the translations about why the font is switching.

Coding standards for `#po:` vs `# po:` although we don't have a set standard, 2/3 of the existing hints omit the space. When adding hints, I've worked on the basis of keeping it consistent within each campaign, for data/gui there aren't many hints at all, but omitting the space has a slight lead in data/gui, in addition to having the lead overall.

Comment (babaissarkar) [reply data/gui/widget/button_success.cfg:250]: Please if the hint message is adequate.

Comment (stevecotton) [reply data/gui/widget/button_success.cfg:250]: Looks good

**Review by stevecotton — APPROVED** [review]

**Review by soliton- — APPROVED** [review]
Works as expected. Thanks!

**Comment by stevecotton** [comment]
Oops, should have squashed that :(

**Pull request merged**